### DISCUSSION TITLE
airisp/docs/flash

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#23] [comment]
# airisp/docs/flash

AirISP支持烧录HEX或`BIN格式的文件到芯片的 FLASH 中。 我们可以使用write_flash命令来执行烧录操作，像下面这样： write_flash 命令参数 write_flash命令有如下参数： 1. --erase-all或者-e，作用是在烧录的时候擦除全部flash，建议添加。 2. --no-progress或者-p，作用是在...

https://arduino.luatos.com/airisp/docs/flash.html

**Comment by pinpang** [comment]
烧写速度好慢……
